### ISSUE TITLE
Dev Test: friction 1.6.0-dev.19

### ISSUE THREAD

**Issue by zepumph** [phetsims/qa#717]
@Matthew-Moore240  @terracoda @emily-phet @jessegreenberg, friction 1.6.0-dev.18 is ready for dev testing. 
The goal of this test is to hammer out the inclusive feature-set and get that ready for publication, but we will not proceed with publication until PhET-iO instrumentation is complete (a couple of weeks away at least). Document issues in https://github.com/phetsims/friction/issues and link to this
issue. Please begin testing right away.

Assigning @kathy-phet and @KatieWoe for prioritization.

<details>
<summary><b>General Dev Test</b></summary>

<h3>What to Test</h3>

- Click every single button.
- If there is sound, make sure it works.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.
- Try to include browser version numbers
- If there is a console available, check for errors and include them in the Problem Description.
- Run through the string tests on at least one platform, especially if it is about to go to rc.
- Check the Game Up harness on one platform.

<h3>Focus and Special Instructions</h3>

[Provide further instructions here. If you have any further tests you want done or specific platforms you want tested,
list them here. Any behaviors you want QA to pay special attention to should be listed here.]

<h3>General Dev Test Platforms</h3>

- [x] Latest macOS, Safari (Time =3.5 hr)
- [x] Windows 10, Chrome (Time = 3 hr )
- [x] Windows 10, Firefox (Time = 2.5 hr)
- [x] Latest iOS, Safari (Time = 2.75 hr)

Light testing, or optionally skip if time crunch: 
- [x] Latest macOS, Chrome (Time = 1.5 hr)
- [x] Latest Chrome OS, Chrome (Time =1.5 hr )

<h3>Issues to Verify</h3>

- [x] [iPad Reset Freeze](https://github.com/phetsims/friction/issues/185) (confirm you can't reproduce just like me)

These issues should have either use the labels "status:ready-for-qa" or "status:ready-for-review." If it is ready for
QA then close the issue if fixed. If ready for review then leave open and assign back to the developer.

<h3>Link(s)</h3>

- **[Simulation](https://phet-dev.colorado.edu/html/friction/1.6.0-dev.19/phet/friction_all_phet.html)**

<hr>

</details>

<details>
<summary><b>Accessibility (a11y) Dev Test</b></summary>

<h3>What to Test</h3>

- Specific instructions can be found below.
- Make sure the accessibility-related feature that is being tested doesn't negatively affect the sim in any way. Here is a list of features to supported in this test: 
  - Alternative Input
  - Interactive Description
  - Sound and Sonification
  - Pan and Zoom
  - Voicing
- Load the a11y view and make sure that interacting with all elements in the simulation updates the appropriate descriptions in the PDOM.

<h3>Focus and Special Instructions</h3>

[Provide further instructions here. If you have any further tests you want done or specific platforms you want tested,
list them here. Any behaviors you want QA to pay special attention to should be listed here.]

<h3>Link(s)</h3>

- **[a11y View](https://phet-dev.colorado.edu/html/friction/1.6.0-dev.19/phet/friction_a11y_view.html)**
- **[Simulation](https://phet-dev.colorado.edu/html/friction/1.6.0-dev.19/phet/friction_all_phet.html)**

<h3>Screen Readers</h3>

This sim supports screen readers. If you are unfamiliar with screen readers, please ask Katie to introduce you to screen
readers. If you simply need a refresher on screen readers, please consult the
[QA Book](https://github.com/phetsims/QA/blob/master/documentation/qa-book.md), which should have all of
the information you need as well as a link to a screen reader tutorial made by Jesse. Otherwise, look over the a11y view
before opening the simulation. Once you've done that, open the simulation and make sure alerts and descriptions work as
intended.

<h3>Platforms and Screen Readers to Be Tested</h3>

- [x] Windows 10 + Latest Chrome + Latest JAWS
- [x] Windows 10 + Latest Firefox + Latest NVDA
- [x] macOS + Safari + VoiceOver
- [ ] **iOS + Safari + VoiceOver (only if specified in testing issue)** Not supported

<h3>Critical Screen Reader Information</h3>

We are tracking known screen reader bugs in
[this Google Document](https://drive.google.com/open?id=12kTs-g-iKEIH1dyG7Q41_W_oNL4gUKbkW-IQgZjMUBw). If you find a
screen reader bug, please check it against this list.

<h3>Keyboard Navigation</h3>

This sim supports keyboard navigation. Please make sure it works as intended on all platforms by itself and with a
screen reader.

<h3>Magnification</h3>

This sim supports magnification with pinch and drag gestures on touch screens, keyboard shortcuts, and mouse/wheel controls. Please test magnfication and make sure
it is working as intended and well with the use cases of the simulation. Due to the way screen readers handle user input, magnification is NOT expected to
work while using a screen reader so there is no need to test this case.

<h3>Final Requests</h3>

- [ ] If this sim is not in [this list](https://phet.colorado.edu/en/accessibility/prototypes) or up to date there, make an
issue in [website](https://github.com/phetsims/website) to ask if PhET research page links need updating. Please assign
to @terracoda and @emily-phet.

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>

  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary>

  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>

  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>

  We typically assign the developer who opened the issue in the QA repository.

  </details>

  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>

  You should:

  1. Consult the [QA Book](https://github.com/phetsims/QA/blob/master/documentation/qa-book.md).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by zepumph** [comment]
Excellent! Thank you @KatieWoe  and all QA.
